openapi.yaml

```diff
@@ -1646,6 +1529,31 @@ paths:
           description: Modified the labels of the SBOM
         '404':
           description: The SBOM could not be found
+  /api/v2/sbom/{id}/license-export:
+    get:
+      tags:
+      - sbom
+      operationId: getLicenseExport
+      parameters:
+      - name: id
+        in: path
+        description: Digest/hash of the document, prefixed by hash type, such as 'sha256:<hash>' or 'urn:uuid:<uuid>'
+        required: true
+        schema:
+          type: string
+      responses:
+        '200':
+          description: license zip file
+          content:
+            application/octet-stream:
```

Comment:
We should change this to the actually returned type.

Reply:
GPT told me that the best way to handle gzip is with body = Vec<u8>, and this is generated via cargo xtask generate-schemas.

Reply:
@bxf12315 please take a look at https://github.com/bxf12315/trustify/pull/7